packages/components/pie-button/src/index.ts

```diff
@@ -56,6 +54,8 @@ export class PieButton extends LitElement {
     static styles = unsafeCSS(styles);
 }
 
+customElements.define('pie-button', PieButton);
```

Comment:
where is `customElements` defined? is that a browser property?

Reply:
yep this is a browser property 😄 https://lit.dev/docs/components/defining/

Reply:
Question: why we had to change this?

Reply:
It needs to be changed for the @custom-elements-manifest/analyzer package which doesn't recognise the shorthand unfortunately - https://custom-elements-manifest.open-wc.org/analyzer/getting-started/ but both mean the same thing 👍